src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -371,6 +379,39 @@ private static void DisplayFullInformation(TestResult result)
             }
         }
 
+        /// <summary>
+        /// Returns the parent Execution id of given test result. 
+        /// </summary>
+        /// <param name="testResult"></param>
+        /// <returns></returns>
+        private Guid GetParentExecutionId(TestResult testResult)
+        {
+            var parentExecutionIdProperty = testResult.Properties.FirstOrDefault(property =>
+                property.Id.Equals(ParentExecutionIdPropertyIdentifier));
+            return parentExecutionIdProperty == null
+                ? Guid.Empty
+                : testResult.GetPropertyValue(parentExecutionIdProperty, Guid.Empty);
+        }
+
+        /// <summary>
+        /// Returns execution id of given test result
+        /// </summary>
+        /// <param name="testResult"></param>
+        /// <returns></returns>
+        private Guid GetExecutionId(TestResult testResult)
+        {
+            var executionIdProperty = testResult.Properties.FirstOrDefault(property =>
+                property.Id.Equals(ExecutionIdPropertyIdentifier));
+            var executionId = Guid.Empty;
+
```

Comment:
Log error if execution id is null

Reply:
@singhsarab I am not sure what you mean here. What behavior should be logged as error? 

The `executionIdProperty` is null often because it regularly is not present in `testResult.Properties`. On the other hand `executionId` is never null because it is a ` Guid`.

Reply:
Yep for xunit it would always be

Reply:
Okay, should I then make any change or keep it as is? We don't always log an error right?

Reply:
I think we don't need to log an error given the context